src/homePage/homePage.component.tsx

```diff
@@ -0,0 +1,324 @@
+  color: '#FFFFFF',
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  backgroundColor: (theme as any).colours?.homePage?.blueButton,
+  '&:hover': {
+    //Check if null to avoid error when loading
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    backgroundColor: (theme as any).colours?.homePage?.blueButton
+      ? alpha(
+          // eslint-disable-next-line @typescript-eslint/no-explicit-any
+          (theme as any).colours?.homePage?.blueButton,
+          0.8
+        )
+      : '#FFFFFF',
+  },
+}));
+
+const baseHomePage = (props: HomePageProps): React.ReactElement => {
+  return (
+    <div id="dg-homepage">
+      <div
+        style={{
+          backgroundImage: `url(${props.backgroundImage})`,
+          backgroundPosition: 'center 40%',
+          width: '100%',
+          height: 250,
+        }}
+      >
+        <div
+          style={{
+            backgroundImage: `url(${props.greenSwirl1Image}), url(${props.decal1Image})`,
+            backgroundRepeat: 'no-repeat',
+            backgroundPosition: 'top left, top right',
+            width: '100%',
+            height: 250,
+          }}
+        >
+          <Box
+            sx={{
+              position: 'relative',
+              left: '50%',
+              top: '45px',
+              transform: 'translate(-50%)',
+            }}
+          >
+            <Typography variant="h2" sx={backgroundTitleStyles}>
+              <strong>Inventory Managment</strong>
+            </Typography>
+            <Typography variant="h2" sx={backgroundTitleStyles}>
+              for <strong>EPAC</strong> facilities
+            </Typography>
+          </Box>
+        </div>
+      </div>
+      <Box
+        sx={{
+          transform: 'translate(0px, -20px)',
+          marginLeft: '8%',
+          marginRight: '8%',
+        }}
+      >
+        <Paper sx={paperStyles} elevation={1}>
+          <Grid container style={{ height: '100%' }}>
+            <Grid item xs={6}>
+              <Box sx={paperContentStyles}>
+                <Typography
+                  variant="h3"
+                  sx={(theme) => ({
+                    fontWeight: 'bold',
+                    fontSize: '32px',
+                    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+                    color: (theme as any).colours?.homePage?.heading,
+                    marginBottom: theme.spacing(2),
+                  })}
+                >
+                  {'Catalogue'}
+                </Typography>
+                <PaperDescription variant="body1">
+                  {
+                    'Browse catalogue categories, sub categories and catalogue items.'
+                  }
+                </PaperDescription>
+                <Box marginTop="16px">
+                  <Button
+                    color="primary"
+                    variant="contained"
+                    component={Link}
+                    to={'/catalogue/*'}
```

Comment:
```suggestion
                    to={'/catalogue'}
```
This doesn't work currently.

Reply:
Could you  also add some e2e tests for the homepage, just for the links e.g. https://github.com/ral-facilities/datagateway/blob/develop/packages/datagateway-dataview/cypress/e2e/homePage.cy.ts